client/src/app.tsx

```diff
@@ -43,9 +43,12 @@ const WritersHomepage = React.lazy(() => import("./writers-homepage"));
 const Sitemap = React.lazy(() => import("./sitemap"));
 const Playground = React.lazy(() => import("./playground"));
 const Observatory = React.lazy(() => import("./observatory"));
+const Debug = React.lazy(
+  () => /* webpackChunkName: "debug" */ import("./ui/molecules/debug")
```

Comment:
@LeoMcA Any idea why this isn't working, and the chunk is called `static/js/7054.chunk.js`?

Reply:
Oh, never mind, I may have found it: e772797734

Reply:
Ah, that did it, now it's called `static/js/debug.chunk.js`.